.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "coresdk/external"]
 	path = coresdk/external
 	url = https://github.com/splashkit/splashkit-external
+[submodule "tools/translator"]
+	path = tools/translator
+	url = https://github.com/alexcu/splashkit-translator
```

Comment:
Will switch to `splashkit/splashkit-translator` once https://github.com/splashkit/splashkit-translator/pull/9 is merged